lib/player.rb

```diff
@@ -0,0 +1,29 @@
+# rubocop:disable Lint/UnreachableCode
+
+class Player
+  attr_accessor :token, :name
```

Comment:
- [ ] You shouldn't use  `attr_accessors` unless you really need to. Using `attr_accessors` means that the property can be written to/changed from anywhere including from outside the class.
screenshot of requirements 👇 
![Screen Shot 2020-08-28 at 1 12 56 AM](https://user-images.githubusercontent.com/48804258/91503751-9c821f80-e8cb-11ea-8c8c-8df1d3348157.png)

Reply:
This part of the comment has not been addressed. Since the player cannot choose the symbol, it is best practice to use `attr_reader`. `attr_reader :token`